model/components/inspectit.yml

```diff
@@ -9,6 +9,7 @@ components:
     logo: inspectit
     categories:
       - agent
+      - instrumentation-lib
```

Comment:
I'm not sure whether we should have both for inspectit. Because, actually there is no standalone inspectIT instrumentation library right? So maybe just leave only agent for inspectIT

Reply:
OK, updating..